Examples/Algorithms/Digitization/include/ActsExamples/Digitization/DigitizationConfig.hpp

```diff
@@ -175,4 +169,5 @@ class DigitizationConfig {
       std::pair<Acts::GeometryIdentifier, std::vector<Acts::BoundIndices>>>
```

Comment:
Actually, I also find this member function weird. Conceptionally, it would make more sense to have it as a free function, since anyways all members are public...

Reply:
I agree that is weird. I just checked and it seems only be used in python bindings. I am not sure if it is safe to be removed?

Reply:
Hmm it seems to be requested by `RootMeasurementWriter`, but not used in it (I did not find any use in the algorithm, even though it is in the config). So if that is true, we should be able to remove it